### ISSUE TITLE
Feature Request: Hot key to make hostile tokens visible in the combat tracker when using the auto hide feature of this module.

### ISSUE THREAD

**Issue by SirAron111** [David-Zvekic/TokensVisible#32]
As stated in the title. i think it would be great if i could change the visibility status of a token with a hotkey when i DM. i like the way the auto hide function works, making my players not suddenly see the 20 zombies hiding in the next room because they show up on the combat tracker. This way i don't need to hide them manually, but they are already in the tracker the moment the player enters the room XD.

Of course, even better than a hotkey or together with a hotkey would be a way to auto reveal hostile tokens if a player has line of sight to it, and it's within his vision range or in a light source the player can see. So i don't even have to use a hotkey.

Call me lazy but as a DM i got a lot of tokens that i would like visible/nonvisible with the module called your tokens visible Xp. k bad joke aside, would appreciate such a feature if possible.

**Comment by David-Zvekic** [comment]
good idea. Thanks for the feedback!

I was thinking about implementing another feature "Party Vision" to allow
all player controlled tokens to see anything that any other player
controlled token with the same token disposition can see.

The technique to implement Party Vision would be required for me to
determine when an enemy becomes visible to "any player" which is the
question that needs to be answered when the combat tracker scrolls to a
hidden combatant to decide if it should reveal the initiative order.

**Comment by SirAron111** [comment]
Sounds interesting. But make sure to make the party vision an optional thing. Because certain systems relay on individual vision for tokens. Not having the same, Intel also promotes RP especially in DnD 5E systems. so it would be good to turn it on or of as needed.

**Comment by David-Zvekic** [comment]
yes Party Vision would be totally optional.  The same way Token Multivision is optional.
